docs/content/learn/tutorials/_index.md

```diff
@@ -13,7 +13,7 @@ The following tutorials provide step-by-step walk-throughs explaining how to cre
 ## [Control Layer Topics](control/)
 
 - [Creating a Basic Control Catalog](control/basic-catalog/): Explains how to create a [catalog](/concepts/terminology/#catalog) of [controls](/concepts/terminology/#control) using the OSCAL [catalog model](/concepts/layer/control/catalog/).
-
+- [Creating a Basic Profile](/learn/tutorials/profile/): A tutorial on creating a baseline using the OSCAL [profile model](/concepts/layer/control/profile/).
```

Comment:
Align the description with the other ones. Suggestion: Explains how to create a selected set of controls from a catalog, also known as baseline, using the OSCAL [profile model](/concepts/layer/control/profile/).